### PR TITLE
Set up CI/CD with GitHub Actions

### PR DESCRIPTION
**Pull request by Nihisil**
Resolves https://github.com/Nihisil/nimble-certification-elixir/issues/4

## What happened

All required GitHub secrets and Heroku variables were set up.

This PR should have all green tests and after merging to develop branch code should be deployed to Heroku.

## Proof Of Work

All tests are green:
image

I manually triggered the deployment action (through VS code GA extension), the action was finished without errors, and the code was deployed to Heroku:
- https://github.com/Nihisil/nimble-certification-elixir/actions/runs/2312179585
- https://aleksei-elixir-ic.herokuapp.com/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,4 +36,7 @@ npm-debug.log
 /assets/node_modules/
 
 # IDE files
-.idea
\ No newline at end of file
+.idea
+
+# Mac OS X files
+.DS_Store
```
Comment (junan) [inline .gitignore:42]: Lets add a new line
```suggestion
.DS_Store

```

**Review by junan — APPROVED** [review]

**Comment by junan** [comment]
@Nihisil I noticed that we have not created the whole backlog(the tasks) yet. We usually create the whole projects tasks first, then start work one by one.  Creating the whole project backlog helps the planning. 

You can check [internal-certification-elixir](https://github.com/nimblehq/internal-certification-elixir) to get the the idea how others did the whole backlog.

File: .gitignore (hunk 1 above)
Comment (Nihisil) [reply .gitignore]: Thanks. For this project, I decided to try VS code (previously I worked with Intellij IDEA), and it turned out that in VS code option "Insert new line on file save" is disabled by default, don't know why they have such a default.

I have enabled this option in IDE and fixed the file as you suggested.

**Comment by Nihisil** [comment]
@junan yep, before creating a full project backlog I was going through the onboarding path, especially through the backlog organization section the last few days. I already started to write the document with the list of required tickets, and plan to finalize it by moving to GitHub projects later today or the next day.

I will continue to work on tickets for this project only after the full backlog will be ready. I just wanted to set up project CI/CD first, to be more familiar with the project template (it is checklist from the onboarding path document as well).

**Review by bterone — APPROVED** [review]

**Pull request merged**